src/lib/utils.ts

```diff
@@ -13,4 +15,16 @@ async function sleep(ms: number): Promise<void> {
   return await new Promise((resolve) => setTimeout(resolve, ms))
 }
 
-export { sleep, removeUndefinedFromObject }
+function addProtocolIfNotPresent(config: Config): Config {
```

Comment:
an util should not be dependent of an external type

```suggestion
function addProtocolIfNotPresent(host: string): string {
```